tests/lib/rules/id-blacklist.js

```diff
@@ -65,6 +65,26 @@ ruleTester.run("id-blacklist", rule, {
             code: "var obj = { key: foo.bar };",
             options: ["f", "fo", "fooo", "b", "ba", "barr", "bazz", "bingg"]
         },
+        {
+            code: "const {foo: bar} = baz",
+            options: ["foo", "bar"],
```

Comment:
@jpramassini 
Hi, In my opinion, this case should be included in `invalid` because it makes a new identifier `bar,` which is in the id-blacklist.

Reply:
Yeah, this should definitely be invalid at some point due to the spirit of the rule. I added that to try and illustrate for now that that's intended behavior for now. Should I make a separate issue to make sure that gets tracked?

Reply:
> Should I make a separate issue to make sure that gets tracked?    

Maybe it would be welcomed, but it is up to you. :)

Reply:
An issue would be great!